panel/chat/langchain.py

```diff
@@ -75,8 +75,17 @@ def _reset_active(self):
         self._active_avatar = self._input_avatar
         self._message = None
 
+    def _on_start(self, kwargs, serialized):
+        model = kwargs.get("invocation_params", {}).get("model_name", "")
+        self._is_streaming = serialized.get("kwargs", {}).get("streaming")
+        messages = self.instance.objects
+        if messages[-1].user != self._active_user:
+            self._message = None
+        if self._active_user and model not in self._active_user:
+            self._active_user = f"{self._active_user} ({model})"
+
     def _stream(self, message: str):
-        if message.strip():
```

Comment:
If token == " ", it will not output the space which leads to this:
![image](https://github.com/holoviz/panel/assets/15331990/efe1ba84-7497-45dd-862e-664b216e25d5)